components/home/Home.js

```diff
@@ -210,6 +210,12 @@ class Home extends Component {
                     </Section> : null
                     }
                 </ScrollView>
```

Comment:
Add Divider to distinguish between the footer and central part of the screen.

Reply:
might need feedback tomorrow during the day to determine what this looks like, not super good with styling